scripts/background.js

```diff
@@ -152,6 +162,110 @@ function FetchSteamUserData( callback )
 	} );
 }
 
+function FetchSteamUserFamilyData( callback )
+{
+	if( userFamilyDataCache !== null )
```

Comment:
This variable is short lived, it should aggressively read from GetLocalOption cache (maybe check by time before sending a request)